app/components/obs/inputs/Components.ts

```diff
@@ -1,16 +1,17 @@
 import Vue from 'vue';
 import * as comps from './index';
 import { TObsType } from './ObsInput';
+import TsxComponent from 'components/tsx-component';
 
 const inputComponents = comps as any as { [key: string]: typeof Vue };
```

Comment:
```suggestion
const inputComponents = comps as any as { [key: string]: typeof TsxComponent };
```

としておくと16行目の `as typeof TsxComponent` を外せそうです